CRM/Utils/System/WordPress.php

```diff
@@ -863,4 +863,26 @@ public function synchronizeUsers() {
     ];
   }
 
+  /**
+   * Send an HTTP Response base on PSR HTTP RespnseInterface response.
+   *
+   * @param \Psr\Http\Message\ResponseInterface $response
+   */
+  public function sendResponse(\Psr\Http\Message\ResponseInterface $response) {
+    // use WordPress function status_header to ensure 404 response is sent
+    status_header($response->getStatusCode());
+    if (function_exists('http_response_code')) {
+      // PHP 5.4+
+      http_response_code($response->getStatusCode());
+    }
+    else {
+      // @todo should we remove this given we are no longer supporting < PHP 5.6 however keeping
+      // in for the moment for compatability with the original AssetBuilder code.
+      header('X-PHP-Response-Code: ' . $response->getStatusCode(), TRUE, $response->getStatusCode());
+    }
```

Comment:
For WP, isn't `status_header()` sufficient? (Guess the E2E tests would tell!) The `http_response_code()` and `header()` formulations feel redundant.